### PR TITLE
Elaborate proposal for LF AI & Data

### PR DESCRIPTION
**Pull request by bowni**
@camillemarini @mattthieu for your review and completion!
(cc @natct10 @ClementMayer @RomainGoussault @jeromechambost)

### CODE REVIEW

File: proposals/substra.md [hunk 1]
```diff
@@ -35,6 +37,10 @@ It is currently under development by the Substra engineering team at Owkin. The
 
 * Please list tools in use by the project:
 
+- [DockerHub](https://hub.docker.com/orgs/substrafoundation)
+- [PyPI](https://pypi.org/project/substra/)
+- <tbc>
```
Comment (natct10) [inline proposals/substra.md]:
```suggestion
- [Helm Charts](https://artifacthub.io/packages/search?page=1&repo=substra)
- <tbc>
```

**Review by natct10 — COMMENTED** [review]
I would suggest to add helm charts

**Comment by bowni** [comment]
> I would suggest to add helm charts

@natct10 can you commit it please? Make things cleaner

**Comment by natct10** [comment]
> > I would suggest to add helm charts
> 
> @natct10 can you commit it please? Make things cleaner

I can't push to this repo, that's why I used the suggestion

File: proposals/substra.md [hunk 2]
```diff
@@ -19,6 +19,8 @@ It is currently under development by the Substra engineering team at Owkin. The
 
 * Statement on alignment with LF AI’s mission:
 
+LF AI & Data's mission includes to drive open source innovation in the AI, ML, DL, and Data domains by enabling collaboration and the creation of new opportunities for all the members of the community. In its effort to enable and foster new privacy-preserinv collaborations on Machine Learning challenges, Substra Framework aligns well and complements other LF AI & Data supported projects in the field of federated learning and privacy-preserving AI.
```
Comment (RomainGoussault) [inline proposals/substra.md]: typo: foster new _privacy-preserinv_ collaborations

File: proposals/substra.md [hunk 3]
```diff
@@ -19,6 +19,8 @@ It is currently under development by the Substra engineering team at Owkin. The
 
 * Statement on alignment with LF AI’s mission:
```
Comment (RomainGoussault) [inline proposals/substra.md:39]: I rephrased some of Origin and history paragraph above:
> ... the main concepts underlying the Substra Framework were designed during the elaboration of a multi-partner research project, named HealthChain, in H2 2017 - H1 2018. This HealthChain project was supported by Bpifrance as part of the Digital Investments Program for the major challenges of the future. This enabled the creation of a consortium ...

**Pull request merged**